NUAttribute.js

```diff
@@ -174,4 +179,17 @@ export default class NUAttribute extends NUObject {
         }
         return null;
     }
+
+    validateNumberValue(attrValue, attrObj) {
+        if (!isNaN(attrObj.minValue) && (isNaN(attrValue) || attrValue < attrObj.minValue)) {
```

Comment:
I think you should check that attrValue is a number indeed I thought there is already number validation. In this case let's add the min/max value check there

Reply:
@natabal we have number type validation within `validateListValues`. we do not have a stand alone number type validation. pls let me know where to plug in these checks